.github/workflows/docker.yml

```diff
@@ -1,19 +1,47 @@
 name: Build and upload Docker image
 
 on:
+  schedule:
+    - cron: "0 23 * * 5"
```

Comment:
releasing on a Friday? 😨 jk I think it's a good idea in general but we're still early on so would prefer manual releases for now

Reply:
yeah like to celebrate the weekend and since by our merge requirement I imagine whatever is there should be working. though I recognize people might not have the celebration feeling towards that😂 Anyways, it is now removed!